src/components/layout/GrayBox/GrayBox.test.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
React 17 allows us to not import React for jsx anymore
Let's remove it and amend eslint and tsconfig files to allow this

Can be useful for typescript - https://stackoverflow.com/a/65539274
Rule for eslint can be disabled

Reply:
Applying the fix from that SO will gives React is undefined. More manual fix is needed.

Reply:
What do you mean `undefined`? I tried to add `jsx: 'react-jsx'` and TS error is gone

Reply:
If needed React can be imported like this `import * as React from 'react';` which is a more recommended way, so I think there should be no problem if you just add `jsx: 'react-jsx'`

Reply:
Please see my latest commit, I have fixed that.